packages/ts/react-crud/src/autoform.tsx

```diff
@@ -345,6 +354,13 @@ export function AutoForm<M extends AbstractModel>({
     setShowDeleteDialog(false);
   }
 
+  const handleKeyDown = (event: KeyboardEvent): void => {
+    if (event.key === 'Enter' && !isSubmitDisabled) {
+      // eslint-disable-next-line no-void
+      void handleSubmit();
```

Comment:
Question: if we change the return type of `handleSubmit` from `Promise<void>` to `Promise<never>`, would it enable us to call it here just as `handleSubmit();` (without the `void` and the hint for the eslint)?

Reply:
I guess not, since we have the following in the catch block:
```tsx
if (error instanceof ValidationError) {
    // Handled automatically
    return;
}
```